.eslintrc.jsdoc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
```

Comment:
JSDoc-specific configuration was pulled out into its own file from `.eslintrc`, but there should be no changes with what was there before.